.gitignore

```diff
@@ -4,4 +4,6 @@ public/lib
 .DS_store
 c4h_keys.txt
 .env
-.package-lock
\ No newline at end of file
+bower_components
+migrations/
+package-lock.json
```

Comment:
Run the command `git rm --cached <file>` to remove package-lock.json from the git index. It still shows up in the PR even though it's been added to gitignore